### DISCUSSION TITLE
RPC is not working when using supabase's docker self-hosted

### DISCUSSION THREAD

**Discussion by J-Light** [mckaywrigley/chatbot-ui#1351] [Help]
Hi,

I am using supabase's docker self-hosted version. The app works fine for the most part, except with files. When using files I notice the following:

```plaintext
"GET /rest/v1/profiles?select=*&user_id=eq.0318c2cc-ce93-40c1-b600-c577d2723409 HTTP/1.1" 200 708 "-" "node"
2024/02/01 01:04:50 [warn] 1109#0: *683891 a client request body is buffered to a temporary file /usr/local/kong/client_body_temp/0000000026, client: 172.20.0.3, server: kong, request: "POST /rest/v1/rpc/match_file_items_openai HTTP/1.1", host: "supabase.example.com"
"POST /rest/v1/rpc/match_file_items_openai HTTP/1.1" 404 226 "-" "node"
```

When using the local supabase, `"POST /rest/v1/rpc/match_file_items_openai"` does not return a `404`. Any ideas why I am getting a 404 for the RPC call database function `match_file_items_openai`?

**Comment by Micheletto** [comment]
@J-Light Did you ever get this working? And if so could you share your fix?

**Reply by J-Light** [reply]
Yes, I did get it working. It was a few hours on weekend trying different things till something clicked. Let me look back and update here when I get some time, possibly this weekend.
